src/background/services/node.ts

```diff
@@ -243,4 +213,24 @@ export default class NodeService extends GenericService {
   async stop() {
 
   }
+
+  async fetch(path: string|null, init: RequestInit): Promise<any> {
```

Comment:
```suggestion
  async fetch(path: string|null, init: RequestInit = { method: 'GET', headers }): Promise<any> {
```

Reply:
Pretty sure this comment needs adjustments. The idea is to create a default header for all requests here.